Avalonia.MusicStore/ViewModels/MusicStoreViewModel.cs

```diff
@@ -92,7 +73,9 @@ private async void LoadCovers(CancellationToken cancellationToken)
             }
         }
 
-        public ReactiveCommand<Unit, AlbumViewModel?> BuyMusicCommand { get; }
-
+        partial void OnSearchTextChanged(string value)
+        {
+            DoSearch(SearchText);
```

Comment:
I'm seeing a warning here. My suggested fix is:

```suggestion
            _ = DoSearch(SearchText);
```